package.json

```diff
@@ -65,7 +65,7 @@
     "react": "*",
     "react-native": "*",
     "react-native-gesture-handler": ">=2.16.1",
-    "react-native-reanimated": ">=3.16.0"
+    "react-native-reanimated": ">=3.16.0 || >=4.0.0-"
```

Comment:
fyi: this `-` at the end is not typo, it is about beta version coverage e.g. `4.0.0-beta.3`